History.md

```diff
@@ -1,5 +1,11 @@
 ## v.NEXT
 
+* The `stylus` package now supports
+  [glob imports](http://stylus-lang.com/docs/import.html#file-globbing) like
+  `@import './folder/*'`.
```

Comment:
Thanks for taking this over.

Globing is currently supported only when imported files import additional files with globing, I think. This is the case I have been solving here. For direct glob import, this should be verified, but I feel one more change has to be done.

I was a bit swamped these days, so I could not check this. If you have time, can you check if direct import works. If it does, then great. :-)

Reply:
I'm currently writing tests for this @mitar so I'll be able to verify this shortly.

Reply:
Oh, great!

Ah, I feel so sorry to just make half of the pull request and dump it on you. I was just not sure if it will be even accepted, so I didn't bother make docs and tests then. But now I haven't yet cycled back to this code to finish it.

Thanks for picking it up.